test/org/sagebionetworks/bridge/services/AppConfigServiceTest.java

```diff
@@ -255,13 +390,29 @@ public void getAppConfigForUserReturnsOldestVersion() {
     @Test
     public void createAppConfig() {
         when(mockStudyService.getStudy(TEST_STUDY)).thenReturn(study);
+        when(mockSchemaService.getUploadSchemaByIdAndRev(any(), any(), anyInt())).thenReturn(mockUploadSchema);
+        when(mockSurveyService.getSurvey(any(), any(), anyBoolean(), anyBoolean())).thenReturn(mockSurvey);
+        when(mockAppConfigElementService.getElementRevision(any(), any(), anyLong())).thenReturn(mockConfigElement);
+        
+        when(mockSurvey.isPublished()).thenReturn(true);
         
         AppConfig newConfig = setupAppConfig();
+        newConfig.setClientData(TestUtils.getClientData());
+        newConfig.setSurveyReferences(ImmutableList.of(new SurveyReference("id", "guid", DateTime.now())));
+        newConfig.setSchemaReferences(ImmutableList.of(new SchemaReference("id", 1 )));
+        newConfig.setConfigReferences(ImmutableList.of(new ConfigReference("id", 1L)));
         
         AppConfig returnValue = service.createAppConfig(TEST_STUDY, newConfig);
+        
         assertEquals(TIMESTAMP.getMillis(), returnValue.getCreatedOn());
         assertEquals(TIMESTAMP.getMillis(), returnValue.getModifiedOn());
         assertEquals(GUID, returnValue.getGuid());
+        assertEquals(newConfig.getLabel(), returnValue.getLabel()); //
+        assertEquals(TEST_STUDY.getIdentifier(), returnValue.getStudyId()); //
+        assertNotNull(returnValue.getClientData());
+        assertNotNull(returnValue.getSurveyReferences());
+        assertNotNull(returnValue.getSchemaReferences());
+        assertNotNull(returnValue.getConfigReferences());
```

Comment:
Instead of checking that they exist, would it be better to check that they match the expected values?

Reply:
Changed these to test actual values